src/translations/es.json

```
@@ -0,0 +1,62 @@
+{
+  "camera": {
+    "all_cameras": "Todas las Vámaras",
+    "cameras": "Cámaras"
+  },
+  "climate": {
+    "all_climates": "Todos los Aires Acondicionados",
+    "climates": "Aires Acondicionados"
```

It's about climate-control (HVAC), not just air conditioning.
See https://www.home-assistant.io/integrations/climate/

```suggestion
    "all_climates": "Todos los Climas",
    "climates": "Climas"
```

---

In Spanish, the word "clima" refers to weather conditions, which would be "weather" in English. In the region where I live, "aire acondicionado" (air conditioning) also controls heating.

In some regions, "climate" is translated as "termostato", which is the device that controls temperature. Similarly, there are other regions where the words "clima" or "climatización" are used.

Therefore, the word "clima" is not incorrectly used, but its meaning depends a lot on the region.

In Home Assistant, the term "termostato" is used when adding an entity to the dashboard.

---

It's more that I've tried to review, but I have absolutely no knowlegde of the spanish language.
I trust on you to apply the correct translations. 😊

Just keep in mind it's about the Home Assistant entities and languages or used in more that one location.

Thank you for your contribution!